mbuild/packing.py

```diff
@@ -189,9 +184,6 @@ def fill_box(
                 )
         if n_compounds is None and box is not None:
             if len(compound) == 1:
-                compound_mass = np.sum(
-                    [a.mass for a in compound[0].to_parmed().atoms]
-                )
                 # Conversion from kg/m^3 / amu * nm^3 to dimensionless units
                 n_compounds = [
                     int(
```

Comment:
In this section, is `compound_mass` expected to be here? One of these is causing the current tests to fail.

Reply:
Thanks for catching that, looks like I just had to change the variable name.